setup.py

```diff
@@ -109,14 +110,15 @@ def get_custom_rc_file(self):
     classifiers=[
         'Development Status :: 2 - Pre-Alpha',
         'Intended Audience :: Developers',
-        'License :: OSI Approved :: APLv2 License',
+        'License :: OSI Approved :: Apache Software License',
```

Comment:
Would be good to confirm. Also here's a list of valid classifiers: https://pypi.python.org/pypi?%3Aaction=list_classifiers